### PR TITLE
Show off QR code monster

### PR DESCRIPTION
**Pull request by igor-davidyuk**
This PR adds a new example notebook showing off the popular model

CVS-123243

### CODE REVIEW

File: notebooks/265-qrcode-monster/265-qrcode-monster.ipynb [hunk 1]
```diff
@@ -0,0 +1,974 @@
+{
```
Comment (apaniukov) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #111.            self.vae_decoder = core.compile_model(vae_decoder)
</pre><p>Why no <code>device</code> here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (apaniukov) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    scheduler = UniPCMultistepScheduler.from_config(pipe.scheduler.config)
</pre><p>This <code>scheduler</code> is not used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (apaniukov) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.        seed: int = 42,
</pre><p>This way you have to use a seed every time. <code>seed: Optional[int] = 42</code>  and <code>if seed is not None: np.random.seed(int(seed))</code> makes it more flexible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (itrushkin) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    %pip install -q "openvino-dev&gt;=2023.1.0"
</pre><p>Can we install openvino package instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: nice catch! fixed

Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: removed!

Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: makes sense! fixed

Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: fixed!

Comment (itrushkin) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    import cv2
</pre><p>This import requires opencv-python to be installed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (itrushkin) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <p>Please remove empty line at the beginning of the cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (itrushkin) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <p>Please remove empty line at the beginning of the cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

**Review by sbalandi — APPROVED** [review]

**Review by apaniukov — APPROVED** [review]

File: notebooks/265-qrcode-monster/265-qrcode-monster.ipynb (hunk 1 above)
Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: added installation of the require package

Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <p>removed</p>

Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <p>removed</p>

File: notebooks/265-qrcode-monster/265-qrcode-monster.ipynb [hunk 2]
```diff
@@ -0,0 +1,971 @@
...
```
Comment (eaidova) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -q accelerate diffusers transformers torch gradio qrcode opencv-python
</pre><p> </p><p>--extra-index-url https://download.pytorch.org/whl/cpu</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (eaidova) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    import qrcode
</pre><p>please add some description before this function or/and move it closer to its usage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (eaidova) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    controlnet_inputs = {
</pre><p>does it really necessary to use 768x768 input for generation? smaller input size requires less memory for conversion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (eaidova) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.        options=[d for d in core.available_devices if "GPU" not in d] + ["AUTO"],
</pre><p>why do you exclude GPU for inference? Stable Diffusion does not have issues for GPU running</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: added

Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: moved the code block

Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: yes, it does. with 512x512 resolution I could not get a readable qr code even with the maximal controlnet guidance, 
see the example

Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: fixed!

Comment (eaidova) [inline notebooks/265-qrcode-monster/265-qrcode-monster.ipynb:1]: <p>there is no any text or code between these 2 headers. I think no need to add both in table of contents</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by eaidova** [comment]
@igor-davidyuk I'm ready to merge, but here hole in numeration, could you please move notebook on 264 position

**Review by eaidova — APPROVED** [review]

File: notebooks/265-qrcode-monster/265-qrcode-monster.ipynb (hunk 2 above)
Comment (igor-davidyuk) [reply notebooks/265-qrcode-monster/265-qrcode-monster.ipynb]: <p>removed</p>

**Pull request merged**